src/newcharacter.cpp

```diff
@@ -997,6 +1012,9 @@ tab_direction set_stats(WINDOW *w, player *u, points_left &points)
         } else if (action == "NEXT_TAB") {
             delwin(w_description);
             return tab_direction::FORWARD;
+        } else if (action == "HELP_KEYBINDINGS") {
+            // Need to redraw since the help window obscured everything.
+            draw_tabs(w, _("STATS"));
         } else if (action == "QUIT" && query_yn(_("Return to main menu?"))) {
```

Comment:
For cases like this (code is already in a different style) should I match the existing code style, change my additions only, or change the style of the whole function?